client/src/modules/stock/lots/modals/edit.modal.html

```diff
@@ -35,6 +35,7 @@
       <bh-currency-input
         label="STOCK.UNIT_COST"
         data-lot-currency-input
+        min="0.00009"
```

Comment:
Where did this number come from?   Doesn't it depend on the currency being used (eg CFs?) and the current exchange rate?  I presume this comes from the minimum bill/coin in CF that is accepted converted to dollars?

Reply:
No, it's more basic than that.  This comes from the fact that we only store up to four decimal places in the `unit_cost` column of the `lot` table.  I wasn't sure if "min" is `>` or `>=` so I went one below it and put `0.00009` instead of `0.0001`.

@lomamech and I discussed this to some extent.  We have a some dangling values that are $0.001 at IMCK for example, but we display them as $0.00 because we only show one two decimal places.  Ultimately, this should probably be user-customizable.  We'd need to update a lot of places though.

Reply:
By the way, with your line of inquiry, you are on the right track. This component normally forces you to pay the minimum currency value, as you are asking about.  I'm telling it to ignore that restriction for the purposes of setting per-pill cost which may be much, much smaller than the minimum amount possible to pay.